[2023-06-22 10:18:45,186] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2022-10-10T00:00:00+00:00 [queued]>
[2023-06-22 10:18:45,191] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2022-10-10T00:00:00+00:00 [queued]>
[2023-06-22 10:18:45,191] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-06-22 10:18:45,191] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-06-22 10:18:45,191] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-06-22 10:18:45,200] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2022-10-10 00:00:00+00:00
[2023-06-22 10:18:45,204] {standard_task_runner.py:52} INFO - Started process 8952 to run task
[2023-06-22 10:18:45,207] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'extrai_dados', 'scheduled__2022-10-10T00:00:00+00:00', '--job-id', '151', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpgegiw01i', '--error-file', '/tmp/tmp42rrbsv_']
[2023-06-22 10:18:45,208] {standard_task_runner.py:80} INFO - Job 151: Subtask extrai_dados
[2023-06-22 10:18:45,244] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.extrai_dados scheduled__2022-10-10T00:00:00+00:00 [running]> on host alura
[2023-06-22 10:18:45,284] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2022-10-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-10T00:00:00+00:00
[2023-06-22 10:18:46,540] {python.py:173} INFO - Done. Returned value was: None
[2023-06-22 10:18:46,544] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=extrai_dados, execution_date=20221010T000000, start_date=20230622T131845, end_date=20230622T131846
[2023-06-22 10:18:46,563] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-06-22 10:18:46,578] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
